test/decoders/test_video_decoder_ops.py

```diff
@@ -426,11 +573,12 @@ def test_video_get_json_metadata_with_stream(self):
         assert metadata_dict["minPtsSecondsFromScan"] == 0
         assert metadata_dict["maxPtsSecondsFromScan"] == 13.013
 
+    # TODO: Not sure whether this test still makes a lot of sense
```

Comment:
Probably not - certainly don't do any work to keep it around. As I pointed out above, we'll need a re-think on metadata in general to make everything fit together. I _think_ we left space for audio in what we have, but we'll find out soon enough.